doc/source/whatsnew/v1.1.0.rst

```diff
@@ -750,6 +750,7 @@ Other
 - Bug in :meth:`DataFrame.__dir__` caused a segfault when using unicode surrogates in a column name (:issue:`25509`)
 - Bug in :meth:`DataFrame.plot.scatter` caused an error when plotting variable marker sizes (:issue:`32904`)
 - :class:`IntegerArray` now implements the ``sum`` operation (:issue:`33172`)
+- Added test for getting values by index value during groupby aggregation (:issue:`33439`)
```

Comment:
can you remove this, we don't add whatsnew for tests

Reply:
sure I removed it, thought I think someone should add a note about that bug being fixed (btw the latest 0.25.x version also had the bug it seems)